src/unix/redox/mod.rs

```diff
@@ -131,6 +131,22 @@ s! {
         pub thousands_sep: *const c_char,
     }
 
+    pub struct msghdr {
+        pub msg_name: *mut c_void,
+        pub msg_namelen: crate::socklen_t,
+        pub msg_iov: *mut crate::iovec,
+        pub msg_iovlen: size_t,
+        pub msg_control: *mut c_void,
+        pub msg_controllen: size_t,
```

Comment:
From the linked source, should this also be `socklen_t`?

Reply:
This has been corrected in relibc with https://gitlab.redox-os.org/redox-os/relibc/-/commit/27f3924126f931f63659f5464f3b1d76a4123e09